app/controllers/api/distributions_controller.rb

```diff
@@ -0,0 +1,17 @@
+# encoding: utf-8
+
+class Api::DistributionsController < Api::BaseController
+  api_versions :v1
+
+  filter_resources_by :series_id
+
+  announce_actions resource: :owner_resource
```

Comment:
Still not sure I follow.  So if this is a `PodcastDistribution`, this controller will announce changes to it with the `Series`?